pkg/fanal/walker/fs_test.go

```diff
@@ -2,8 +2,11 @@ package walker_test
 
 import (
 	"errors"
+	"golang.org/x/exp/slices"
```

Comment:
```suggestion
	"slices"
```

By the way, we use "golang.org/x/exp/maps" and "golang.org/x/exp/slices" in many places, but we can already switch to the standard slices and maps packages.

Reply:
Thanks. Yes, I usually use `slices`, but this PR was open on Sep 13, 2023 (and waiting for Aqua's review for a long time). Trivy was using Go 1.20, which doesn't have the `slices` package. I'll update it.